tools/n64tool.c

```diff
@@ -428,11 +453,11 @@ int main(int argc, char *argv[])
 			return print_usage(argv[0]);
 		}
 	}
-	else if((total_bytes_written % 512) != 0)
+	else if(((total_bytes_written+HEADER_SIZE) % PAD_ALIGN) != 0)
 	{
-		/* Pad size to 512 bytes to avoid possible incompatibilities with 64drive
+		/* Pad size to 16384 bytes 512 bytes to avoid possible incompatibilities with 64drive

```

Comment:
This looks like a typo, the old text should probably be removed.

Reply:
Done